src/core/http.ts

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable no-unused-vars */
+enum HTTPMethods {
+      GET = 'GET',
+      PUT = 'PUT',
+      POST = 'POST',
+      DELETE = 'DELETE'
+}
+
+function queryStringify(data: any): string {
+  if (typeof data !== 'object') {
+    throw new Error('Error');
+  }
+  const arr = ['?'];
+  Object.keys(data).forEach((key) => arr.push(`${key}=${data[key]}&`));
+  return arr.join('').slice(0, -1);
```

Comment:
Выглядит довольно сложно, я бы упростил:
```suggestion
return "?" + Object
 .entries(data)
 .map(([key, value]) => `${key}=${value}`)
 .join("&")
```